packages/simcore-sdk/src/simcore_sdk/config/http_clients.py

```diff
@@ -0,0 +1,34 @@
+from typing import Optional
+
+from pydantic import BaseSettings, Field
+
```

Comment:
what is the difference between this and ``packages/models-library/src/models_library/settings/http_clients.py``??

As far as I recall, there was already a dependency of ``simcore_sdk`` to ``models_library``?

Reply:
good catch, fixed.

Reply:
how did you fix it?

Reply:
i ee the now... sorry! :-)